Userland/Applications/Piano/main.cpp

```diff
@@ -39,7 +39,6 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
 
     auto audio_loop = AudioPlayerLoop::construct(track_manager, need_to_write_wav, wav_writer);
     audio_loop->enqueue_audio();
-    audio_loop->enqueue_audio();
```

Comment:
This will lead to glitchy audio, because there will be a short time during sample computation where no audio plays. Please re-add this line.

Reply:
Oh good catch and sorry for not calling out the assumption I originally made here (that it was a duplicate line error), will update!